packages/better-auth/src/plugins/custom-session/custom-session.test.ts

```diff
@@ -61,6 +61,7 @@ describe("Custom Session Plugin Tests", async () => {
 		const session = await auth.api.getSession({ headers });
 		const s = await client.getSession({ fetchOptions: { headers } });
 		expect(s.data?.newData).toEqual({ message: "Hello, World!" });
+		// @ts-expect-error kinda puzzled about this
```

Comment:
maybe you break some inferXXX type

Reply:
Yeah, this one test is currently broken, still trying to figure out why this broke.